documentation/modules/security/con-certificate-authorities.adoc

```diff
@@ -26,3 +26,15 @@ You can configure the validity period of:
 
 * Cluster CA certificates in `Kafka.spec.clusterCa.validityDays`
 * Client CA certificates in `Kafka.spec.clientsCa.validityDays`
+
+== Options for installing your own CA certificates
+
+In your security model, client applications might need to trust certificates that were issued by an external CA, such as your company's internal CA or a public CA, in order to connect to external systems. {ProductName} provides two methods for configuring your Kafka client applications to connect to the brokers in your Kafka cluster using your own certificates.
+
+You can install your own certificates at two levels:
+
+* At a cluster level--see xref:installing-your-own-ca-certificates-{context}[].
+
+* For each listener (TLS and external) in a Kafka broker--see xref:proc-installing-certs-per-listener-{context}[].
```

Comment:
TLS and external or TLS or external?

Reply:
Good spot. I've reworded as follows:

"For each listener (TLS or external listener, or both) in a Kafka broker..."